src/main/java/edu/umich/its/cpm/MigrationTaskService.java

```diff
@@ -1003,7 +1003,10 @@ private StringBuffer preMigrationChecks(StringBuffer itemStatus,
 				// return AsyncResult
 				return new AsyncResult<String>(setUploadJobEndtimeStatus(id, status));
 			}
-			fileName= replaceDotsInFileNameExceptFileExtention(fileName);
+			// replacing dots with _ don't need for Weblinks https://itsjira.umms.med.umich.edu/browse/TLCPM-653
+			if(!type.equals(Utils.CTOOLS_RESOURCE_TYPE_URL)) {
```

Comment:
The code is already is been like that, i will try to add as much of comments as possible now
Ctools weblinks are same as Box weblinks. it's the naming convention that is followed in the code as weblink are stored in the Migration_Box_file table after getting it from ctools.